redash/handlers/queries.py

```diff
@@ -18,6 +20,34 @@
 from redash.utils import collect_parameters_from_request
 
 
+# Ordering map for relationships
+order_map = {
+    'name': 'lowercase_name',
+    '-name': '-lowercase_name',
+    'runtime': 'query_results-runtime',
+    '-runtime': '-query_results-runtime',
+    'executed_at': 'query_results-retrieved_at',
+    '-executed_at': '-query_results-retrieved_at',
+    'created_by': 'users-name',
+    '-created_by': '-users-name',
+}
+
+
+def order_results(results, default_order='-created_at'):
+    """
+    Orders the given results with the sort order as requested in the
+    "order" request query parameter or the given default order.
+    """
+    # See if a particular order has been requested
+    order = request.args.get('order', '').strip() or default_order
+    # and if it matches a long-form for related fields, falling
+    # back to the default order
+    selected_order = order_map.get(order, order)
+    # The query may already have an ORDER BY statement attached
+    # so we clear it here and apply the selected order
+    return sort_query(results.order_by(None), selected_order)
```

Comment:
Interesting. What would happen without this?

Reply:
By default, the `all_queries` class method sorts by `created_at`, which means we just reset it here to be able to apply the sorting according to the query parameter.

Reply:
But isn't SQLA smart enough to just replace the `order_by` when called again?